sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c

```diff
@@ -162,10 +162,8 @@ void _remove_tentative_addr(gnrc_netif_t *netif, const ipv6_addr_t *addr)
         /* Cannot use target address as personal address and can
          * not change hardware address to retry SLAAC => use purely
          * DHCPv6 instead */
-        /* TODO: implement IA_NA for DHCPv6 */
-        /* then => tgt_netif->aac_mode |= GNRC_NETIF_AAC_DHCP; */
-        DEBUG("nib: would set interface %i to DHCPv6, "
-              "but is not implemented yet", netif->pid);
+
+        netif->ipv6.aac_mode |= GNRC_NETIF_AAC_DHCP;
```

Comment:
Don't forget to clear the SLAAC flag, and that DHCPv6 may not be compiled in:

```suggestion
        if (IS_USED(MODULE_DHCPV6_CLIENT_IA_NA) {
            netif->ipv6.aac_mode &= ~GNRC_NETIF_AAC_SLAAC;
            netif->ipv6.aac_mode |= GNRC_NETIF_AAC_DHCP;
        }
        else {
            DEBUG("nib: would set interface %i  to DHCPv6, "
                  "but DHCPv6 is not provided", netif->pid);
        }
```

Reply:
Also, shouldn't `dhcpv6_client_req_ia_na()` be called here as well?

Reply:
> Also, shouldn't `dhcpv6_client_req_ia_na()` be called here as well?

I added it in cba8a2b4244f48033984404dc93b6f8ef91b17e9.

I also used `GNRC_NETIF_AAC_AUTO` as that is the one enum value used for any kind of autoconfiguration at the moment – or is there maybe the need to differentiate between autoconfiguration types?

Reply:
Jepp, sorry quoted "GNRC_NETIF_AAC_SLAAC" purly from memory ;-).